Fun Commands/Mini-Games/Dyno-RPG/treat.txt

```diff
@@ -1,4 +1,11 @@
+{delete}
+{silent}
+{require: Adventurer}
 {!role $1 -Infected}
 {!role $1 -Bleeding}
 {dm:$1} Looks like someone is looking out for you in this forsaken world... 
-You have been cured of all ailments. 
\ No newline at end of file
+You have been cured of all ailments.
+https://media1.tenor.com/images/b92c9b3d1808a5abc588435aae052eb8/tenor.gif?itemid=11061677
+{!announce:#world-gates}
```

Comment:
Again, not sure if the DM announce combo is going to work here.

Reply:
Fair enough. Removed the DM part